src/components/DataRoom/WorldGDP/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { Typography } from '@mui/material'
+import type { BaseBlock } from '@/components/Home/types'
+import LinksWrapper from '@/components/DataRoom/LinksWrapper'
+import css from './styles.module.css'
+import MotionContainer from './MotionContainer'
+import { useSafeDataRoomStats } from '@/hooks/useSafeDataRoomStats'
+import { formatCurrency } from '@/lib/formatCurrency'
+
+type ExtendedBaseBlock = BaseBlock & {
+  subtitle: string
+}
+
+const VALUE_LOCKED_FALLBACK = 68583703689
+const PERCENTAGE_FALLBACK = 0.000679
+
+const WorldGDP = ({ title, subtitle, text, link }: ExtendedBaseBlock) => {
```

Comment:
No need to create an extra type IMO

```suggestion
const WorldGDP = ({ title, subtitle, text, link }: BaseBlock & { subtitle: string }) => {
```